tests/server/test_server_interactions.py

```diff
@@ -975,7 +975,8 @@ async def test_manual_structured_content(self):
         @mcp.tool
         def f() -> ToolResult:
             return ToolResult(
-                content="Hello, world!", structured_content={"message": "Hello, world!"}
+                content=[TextContent(type="text", text="Hello, world!")],
```

Comment:
Here for illustration, instead of making it easy for a user to enhance their "normal" result with a piece of structured data, they would have to load and learn an MCP type.